README.rst

```diff
@@ -21,12 +24,38 @@ Install with pip::
 
     $ pip install flake8-coding
 
-Further options
-----------------
+Options
+-------
+
+``accept-encodings``
+~~~~~~~~~~~~~~~~~~~~
+
+A comma-separated list of acceptable source code encodings for the ``coding:``
+magic comments in files. Default is ``latin-1, utf-8``.
+
+You can pass this as a command-line argument to ``flake8``, e.g.
+``--accept-encodings=utf-8,utf-16``, or put it in your config file, e.g.:
+
+.. code-block:: ini
```

Comment:
Oh, sorry. I don't know that.

Does this error come from that pygments is not installed?

>warning: check: Could not finish the parsing.

If true, could you add pygments into `.travis.yml`?
Then it might be resolved.